packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -488,25 +545,17 @@ export class BuildService {
 
     const clientHost = this.configService.get(CLIENT_HOST_VAR);
 
-    const project = await this.prisma.project.findUnique({
-      where: {
-        id: resource.projectId,
-      },
-    });
-
     const url = `${clientHost}/${project.workspaceId}/${project.id}/${resource.id}/builds/${build.id}`;
 
     return this.actionService.run(
       build.actionId,
-      PUSH_TO_GIT_STEP_NAME(EnumGitProvider[gitOrganization.provider]),
-      PUSH_TO_GIT_STEP_MESSAGE(EnumGitProvider[gitOrganization.provider]),
+      PUSH_TO_GIT_STEP_NAME(EnumGitProvider[gitSettings.gitProvider]),
+      PUSH_TO_GIT_STEP_MESSAGE(EnumGitProvider[gitSettings.gitProvider]),
```

Comment:
In the non-demo-repo case, those lines will now be mapped to `gitProviderArgs.provider` and not the previous `gitOrganization.provider` (perhaps they are essentially the same, I just notices the potential logic change here compared to the original code)

Reply:
look ok